### ISSUE TITLE
type 'int' is not a subtype of type 'String'

### ISSUE THREAD

**Issue by gukeming** [fluttercommunity/flutter_webview_plugin#491]
## System info

Issue occurs on:  /Android 
Plugin version: 0.3.5
Flutter doctor output:

```
[✓] Flutter (Channel stable, v1.7.8+hotfix.2, on Mac OS X 10.13.6 17G65, locale zh-Hans-CN)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 9.2)
[✓] iOS tools - develop for iOS devices
[✓] Android Studio (version 3.3)
[✓] VS Code (version 1.36.1)
[✓] Connected device (3 available)

• No issues found!
```

## Steps to Reproduce
_onHttpError.add(WebViewHttpError(call.arguments['code'], call.arguments['url']));

native code type is int, but WebViewHttpError  code type is String.

error: type 'int' is not a subtype of type 'String'

**Comment by rodrigoavalente** [comment]
Hey, I've encontered the same problem and related it on issue #444, but I didn't know how to handle it (sorry, zero knownledge on creating flutter plugins). Can I install the plugin from your repository and with your adjustment and it will work from scratch? Or I need to do something else?

Thanks.

**Comment by charafau** [comment]
fixed in 0.3.6
